test/topology/util.py

```diff
@@ -469,23 +469,33 @@ async def new_coordinator_elected():
 
     await wait_for(new_coordinator_elected, deadline=deadline)
 
+async def create_new_test_keyspace(cql: Session, opts, host=None):
+    """
+    A utility function for creating a new temporary keyspace with given
+    options.
+    """
+    keyspace = unique_name()
+    # Use CREATE KEYSPACE IF NOT EXISTS as a workaround for
+    # https://github.com/scylladb/python-driver/issues/317
+    await cql.run_async(f"CREATE KEYSPACE IF NOT EXISTS {keyspace} {opts}", host=host)
+    return keyspace
+
 @asynccontextmanager
-async def new_test_keyspace(cql, opts, host=None):
+async def new_test_keyspace(manager: ManagerClient, opts, host=None):
     """
     A utility function for creating a new temporary keyspace with given
     options. It can be used in a "async with", as:
         async with new_test_keyspace(cql, '...') as keyspace:
```

Comment:
new_test_keyspace(ManagerClient, '...') as keyspace

Reply:
fixed in v2